server/sql/schema.sql

```diff
@@ -24,6 +24,7 @@ CREATE TABLE "raid_instance" (
 );
 
 ALTER TABLE "raid_instance" ADD CONSTRAINT "raid_instance_fk0" FOREIGN KEY ("raid_id") REFERENCES "raid"("raid_id")
+ALTER TABLE "raid_instance" ADD CONSTRAINT "raid_instance_fk0" FOREIGN KEY ("user_id") REFERENCES "user"("user_id")
```

Comment:
```suggestion
```
this seems duplicated

Reply:
i dont think so user_id and raid_id are both different foreign keys unless you dont need both line 26 & 27

Reply:
change raid_instance_fk0 to raid_instance_fk1